examples/appdir/src/app/sink/page.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import { Inter } from "next/font/google";
+import { UploadButton, UploadDropzone } from "@uploadthing/react";
+import type { OurFileRouter } from "../_uploadthing";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export default function Home() {
+  return (
+    <main className="flex min-h-screen flex-col items-center justify-center  gap-16 p-24">
+      <div className="flex flex-col items-center justify-center gap-4">
+        <div className="flex gap-4">
+          <UploadButton<OurFileRouter>
```

Comment:
looks like you lost a title 

```
<span className="text-4xl font-bold text-center">
  {`Upload a file using a button:`}
</span>
```

Reply:
I think you're overestimating how much I care about this file lol